k-distribution/tutorial/2_languages/1_simple/1_untyped/simple-untyped.md

```diff
@@ -1,114 +1,129 @@
-// Copyright (c) 2014-2019 K Team. All Rights Reserved.
+---
+copyright: Copyright (c) 2014-2020 K Team. All Rights Reserved.
+---
 
-/*!
-\title{SIMPLE --- Untyped}
-\author{Grigore Ro\c{s}u and Traian Florin \c{S}erb\u{a}nu\c{t}\u{a}
-        (\texttt{\{grosu,tserban2\}@illinois.edu})}
-\organization{University of Illinois at Urbana-Champaign}
-*/
+SIMPLE — Untyped
+================
 
-/*@ \section{Abstract}
-This is the \K semantic definition of the untyped SIMPLE language.
+Author: Grigore Roșu (grosu@illinois.edu)  
+Organization: University of Illinois at Urbana-Champaign
+
+Author: Traian Florin Șerbănuță (traian.serbanuta@unibuc.ro)  
+Organization: University of Bucharest
```

Comment:
This information can go in the markdown YAML header above (where the copyright is). Your choice, but that makes it more scrape-able.

Reply:
I like the idea. Do you have a specific format in mind? Because I can't seem to find a configuration that I like. Apparently I can't put two values for the same tag.